src/Draggable/DragEvent/tests/DragEvent.test.js

```diff
@@ -12,51 +12,51 @@ import {
 
 describe('DragEvent', () => {
   describe('#constructor', () => {
-    test('should be instance of DragEvent', () => {
+    it('should be instance of DragEvent', () => {
```

Comment:
Why are we changing to `it` ? I think `test` is the official way, and `it` is just an alias to `test`. I am guessing the `prettier:shopify/jest` config does this?

https://facebook.github.io/jest/docs/en/api.html#testname-fn-timeout

Reply:
oui - it auto changed it to `it` - i will see if i can get some answers in the `fed` channel

Reply:
@tsov asked - the answer is that it reads more like a sentence:

`it('puts the lotion in the basket')`

Reply:
😆 let's do it